### PR TITLE
Update Chinese

### PR DESCRIPTION
**Pull request by moyuguguji**
Update outdated and missing translations

### CODE REVIEW

File: src/main/resources/assets/spectrum/lang/zh_cn.json [hunk 1]
```diff
@@ -1,1818 +1,3730 @@
...
+    "item.spectrum.suspicious_brew.tooltip.preview": "酒精度越高，持续时间越长 %",
+    "item.spectrum.suspicious_brew.tooltip.preview2": "效果取决于使用的花朵",
+    "item.spectrum.jade_wine": "玉酒",
+    "item.spectrum.jade_wine.tooltip.bloominess": "%d %% 花开度",
+    "item.spectrum.jade_wine.tooltip.sweetened": "加糖",
+    "item.spectrum.jade_wine.tooltip.preview3": "酒精含量越高，正面效果越多、时间越长。 %",
+    "item.spectrum.jade_wine.tooltip.preview": "用玉藤种子和花瓣混合酿造",
+    "item.spectrum.jade_wine.tooltip.preview2": "花瓣比玉藤种子发酵慢，但减少了负面影响",
+    "item.spectrum.pure_alcohol": "高纯度酒精",
+
```
Comment (Noaaan) [inline src/main/resources/assets/spectrum/lang/zh_cn.json]: This is missing. The joke here is a reference to Tabletop RPG's like Dungeons and Dragons, where your Intelligence Modifier is about the regular limit (usually +5)

Comment (SmallYii) [reply src/main/resources/assets/spectrum/lang/zh_cn.json]: Thanks for the correction. I have another question for you. Is there anything special about the glass_crest series equipment? Is it an upgraded version of the malachite equipment? I don't know how to translate it correctly.

Comment (DaFuqs) [reply src/main/resources/assets/spectrum/lang/zh_cn.json]: > Is it an upgraded version of the malachite equipment

yes

Comment (SmallYii) [reply src/main/resources/assets/spectrum/lang/zh_cn.json]: > > Is it an upgraded version of the malachite equipment
> 
> yes

Why is it called glass_crest? Does it have any special meaning?

**Comment by DaFuqs** [comment]
That's a LOT of lang!
How you you feel is the state of the translation at the moment? Could the current state be merged and tested ingame?

**Comment by SmallYii** [comment]
> That's a LOT of lang! How you you feel is the state of the translation at the moment? Could the current state be merged and tested ingame?

There are still more than 500 lines that have not been translated. My network problems have delayed the translation progress. Sorry.

**Comment by moyuguguji** [comment]
> 语言太多了！ 您觉得目前的翻译状态如何？当前状态可以合并并在游戏中测试吗？

It can be merged for testing at the moment.,as long as the changes to the Patchouli book are not significant, there should be no problem.

**Comment by DaFuqs** [comment]
I tested the current lang ingame and it worked wonderfully, not a single formatting error in sight.
I will merge your current version, so your hard work will  be included in the final release, but feel free to do more down the line ✅

**Pull request merged**